api/data_explorer/controllers/facets_controller.py

```diff
@@ -13,17 +12,6 @@
 from data_explorer.util.dataset_faceted_search import DatasetFacetedSearch
 
 
-def _is_histogram_facet(facet):
```

Comment:
Same here, is this used somewhere apart from facets_controller?

Reply:
Used in `deserialize` which is used by `export_url_controller`.